src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -613,7 +614,7 @@ export class SearchInterface extends RootComponent implements IComponentBindings
     new HistoryController(this.element, historyControllerEnvironment);
   }
 
-  private setupDebugInfo() {
+  private async setupDebugInfo() {
```

Comment:
Interesting. Is this needed though if you are not using an `await`?

Reply:
No, I forgot to remove it as I was changing stuff around. Good catch.